aten/src/ATen/native/native_functions.yaml

```diff
@@ -6728,27 +6728,27 @@
   use_c10_dispatcher: full
   variants: method, function
 
-- func: quantile.scalar_out(Tensor self, float q, int? dim=None, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
+- func: quantile.scalar_out(Tensor self, float q, int? dim=None, str interpolation='linear', bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
@mruberry Do we have a preference for whether dim and keepdim should be next to each other?

Reply:
It would make conceptual sense if they were next to each other. Let's implement a NumPy-like signature for the moment (keeping out at the end to be consistent with PyTorch). In the future I think we should consider making these args kwarg-only.

Reply:
The NumPy's signature of quantile function is below:
`numpy.quantile(a, q, axis=None, out=None, overwrite_input=False, interpolation='linear', keepdims=False)`.

The current PR follows the NumPy style, by putting `interpolation` in front of `keepdims`.

I am a little bit uncertain about whether we should make `dim` and `keepdim` next to each other for the moment
Please kindly let me know if the order of `interpolation` and `keepdims` parameters should be exchanged, I will update this PR accordingly. @mruberry @heitorschueroff

Reply:
Let's keep it this way then for the moment.